packages/flutter/lib/src/material/about.dart

```diff
@@ -11,18 +11,18 @@ import 'package:flutter/foundation.dart';
 import 'app_bar.dart';
 import 'debug.dart';
 import 'dialog.dart';
-import 'drawer_item.dart';
 import 'flat_button.dart';
 import 'icon.dart';
 import 'icon_theme.dart';
 import 'icon_theme_data.dart';
+import 'list_tile.dart';
 import 'page.dart';
 import 'progress_indicator.dart';
 import 'scaffold.dart';
 import 'scrollbar.dart';
 import 'theme.dart';
 
-/// A [DrawerItem] to show an about box.
```

Comment:
I have changed the name to AboutListTile (I couldn't come up with something better).